server/routes.go

```diff
@@ -191,6 +191,10 @@ func (s *Server) GenerateHandler(c *gin.Context) {
 			CreatedAt:  time.Now().UTC(),
 			Done:       true,
 			DoneReason: "load",
+			Metrics: api.Metrics{
+				LoadDuration:  checkpointLoaded.Sub(checkpointStart),
```

Comment:
suggestion: it's minor, but this could be calculated once, for example the following with the other use replaced as well, same for the other case.
```go
	loadDuration := time.Since(checkpointStart)

	// load the model
	if req.Prompt == "" {
		c.JSON(http.StatusOK, api.GenerateResponse{
			Model:      req.Model,
			CreatedAt:  time.Now().UTC(),
			Done:       true,
			DoneReason: "load",
			LoadDuration: loadDuration,
			TotalDuration: loadDuration,
		})
		return
	}

...
			if r.Done {
				res.TotalDuration = time.Since(checkpointStart)
				res.LoadDuration = loadDuration
			}
```